distribution/lib/Standard/Base/0.0.0-dev/src/Data/Range.enso

```diff
@@ -107,11 +151,12 @@ type Range
              10.up_to 100 . all (> 5)
     all : (Number -> Boolean) -> Boolean
     all predicate =
-        it start end = if start==end then True else
-            r = predicate start
-            if r then (@Tail_Call it start+1 end) else False
-        res = it this.start this.end
-        res
+        go end_condition current end =
```

Comment:
can we use `find` here to at least remove one duplicate!

Reply:
Replaced with `this . exists (predicate >> .not) . not` as in Vector.